data_analysis/agora/notebooks/preprocessing/preprocessing_utils.py

```diff
@@ -67,10 +67,60 @@ def filter_hasgs(df: pd.DataFrame, chromosome_name_column: str) -> pd.DataFrame:
     regex = re.compile("^(\d|X|Y|MT)")
     keep = df[chromosome_name_column].apply(
         # Keep rows if they have a numerical chromosome name, or have X, Y, or MT
-        lambda row: re.match(regex, row) is not None
-        if isinstance(row, str)
-        else True  # Always true for numbers
+        lambda row: (
+            re.match(regex, row) is not None if isinstance(row, str) else True
+        )  # Always true for numbers
     )
 
     df_filt = df.copy().loc[keep].reset_index(drop=True)
     return df_filt
+
+
+def r_query_biomart() -> pd.DataFrame:
```

Comment:
This is code from the pre-processing notebook that I moved from the notebook to here as an unused function, instead of deleting it.